smartsim/_core/_install/builder.py

```diff
@@ -455,10 +470,10 @@ def build_from_git(self, git_url: str, branch: str, device: str = "cpu") -> None
         # get RedisAI dependencies
         dep_cmd = [
```

Comment:
could we pull the right hand side of this out into a function and avoid some minor duplication?

```
dep_cmd = create_git_build_cmd(self.fetch_pt, self.fetch_tf, self.fetch_onnx, verbose=True)
build_cmd = create_git_build_cmd(self.torch, self.tf, self.onnx)

Reply:
I like it! It also removes the need for the `_TBoolInt` type alias. Will add!